### DISCUSSION TITLE
Kalenderformatierung

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2404] [Q&A]
Hallo,
wäre es möglich das Widget anders zu formatieren?
Ich würde gerne 
- das Datum an einer anderen Stelle darstellen (z.B. vor der Uhrzeit)
- den Tag "Do.," in "Donnerstag," umbenennen 
- die Angabe "vor einem Tag" bzw. "in einem Tag" / "in etwa 3 Stunden" etc. ausblenden können
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/3b0424dd-b674-46e5-bc8a-ff95fa505d0c)

Ist das bzw. wie ist das umsetzbar?
Viele Grüße

**Comment by mcuiobroker** [comment]
Würde dann direkt ein Feature Request einstellen. So kann man da nichts machen. Sind durchgehende Zeilen.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/96c46910-5ed9-4e7f-96c7-49b69f30a365)

**Reply by tschortschde** [reply]
Danke, habe ein Issue eröffnet. Ist die Vorgehensweise so richtig? 😉

**Reply by mcuiobroker** [reply]
Passt schon.
